src/js/ripe.js

```diff
@@ -693,6 +705,26 @@ Ripe.prototype._updateDrag = function(target, position, animate, single, callbac
     mustPreload && preload(this.options.useChain);
 };
 
+Ripe.prototype.getFrames = function(callback) {
+    if (this.config === undefined) {
+        this.getConfig(function(config) {
+            this.config = config;
+            this.getFrames(callback);
+        });
+    }
+
+    var frames = {};

```

Comment:
update the frames references on current documentation

Reply:
Done on https://github.com/ripe-tech/ripe-sdk/pull/3